graphistry/dep_manager.py

```diff
@@ -0,0 +1,30 @@
+import importlib
+
+class DepManager:
+    def __init__(self):
+        self.pkgs = {}
+
+    def __getattr__(self, pkg:str):
```

Comment:
instead call this `def import(...)` or something to make it explicit that code is running